.env.example

```diff
@@ -54,6 +54,9 @@ VITE_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 VITE_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
 
 SHIB_EMULATE=true
+SHIB_AUTH_FIELD=unique_id
```

Comment:
authfield should probably be email to better match how we run it in production. The next line shouldn't matter either if shib_emulate is true? But if it made things work I guess leave it.